src/backend/gl/src/lib.rs

```diff
@@ -127,7 +130,51 @@ impl Share {
     }
 }
 
-pub struct PhysicalDevice(Rc<Share>);
+/// Single-threaded `Arc`.
+/// Wrapper for `Arc` that allows you to `Send` it even if `T: !Sync`.
+/// Yet internal data cannot be accessed in the thread where it was created.
+pub struct Starc<T: ?Sized> {
```

Comment:
nice concept!

Reply:
Yeah. Except it should be `outside of the thread where it was created` 😄

Reply:
Except it should be
`Yet internal data cannot be accessed outside of the thread where it was created.`